src/api/types.ts

```diff
@@ -432,3 +432,9 @@ export interface SystemVendor {
   productName?: string;
   manufacturer?: string;
 }
+
```

Comment:
this file is autogenerated, put it somewhere else for the time being please

Reply:
moved away for now